data.py

```diff
@@ -1,19 +1,23 @@
+from typing import Tuple
+
 import numpy as np
 
 
 class Data:
-    def __init__(self):
-        self.x = []
-        self.t = []
-    
-    def addData(self, n):
-        for i in range(n):
-            self.x.append(np.random.rand(1))
-            self.t.append(np.sin(2*np.pi*self.x[-1]) + np.random.normal(0, 0.1))
-    
-    def getData(self):
-        return self.x
+    def __init__(self, mean: float = .0, std: float = .01):
+        self.mean = mean
+        self.std = std
+
+        self.x = np.empty((0, 1))
+        self.y = np.empty((0, 1))
     
-    def getTarget(self):
-        return self.t
+    def add(self, n: int):
+        x = np.random.rand(n, 1)
+        y = np.sin(2 * np.pi * x) + np.random.normal(self.mean, self.std, size=(n, 1))
 
+        self.x = np.concatenate((self.x, x))
+        self.y = np.concatenate((self.y, y))
+    
+    def get(self) \
```

Comment:
Anyway, every input requires both `x` and `y`, so you don't need to create two methods to return.